Sources/TensorFlow/Operators/LinearAlgebra.swift

```diff
@@ -196,4 +238,26 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
     func qrDecomposition(fullMatrices: Bool = false) -> (q: Tensor<Scalar>, r: Tensor<Scalar>) {
         _Raw.qr(self, fullMatrices: fullMatrices)
     }
+
+    /// Returns the singular value decompositions of one or more matrices.
+    ///
+    /// Computes the SVD of each inner matrix in the `input` tensor such that
+    /// `input[..., :, :] = u[..., :, :] * diag(s[..., :, :]) * transpose(v[..., :, :])`
+    ///
+    /// - Parameters:
+    ///   - computeUv: If `true`, left and right singular vectors will be computed and
+    ///     returned as `u` and `v`, respectively. Otherwise, only the singular values will be
+    ///     computed, which can be significantly faster.
+    ///   - fullMatrices:  If `true`, compute full-sized `u` and `v`. If `false`, compute only the
+    ///     leading `min(shape[rank - 1], shape[rank - 2])` singular vectors. Ignored if
+    //      `computeUv` is `false`.
+    @inlinable
+    func svd(computeUv: Bool = true, fullMatrices: Bool = false
+    ) -> (s: Tensor<Scalar>, u: Tensor<Scalar>?, v: Tensor<Scalar>?) {
+        let (s, u, v) = _Raw.svd(self, computeUv: computeUv, fullMatrices: fullMatrices)
```

Comment:
Is it possible to return the output of `_Raw.svd` directly? That simplifies the body to one line.
```suggestion
        return _Raw.svd(self, computeUv: computeUv, fullMatrices: fullMatrices)
```
The `if !computeUv { ... }` seems redundant to me, if `_Raw.svd` returns `nil` values for `u` and `v` when `computeUV` is `false`.

Reply:
Edit: the current logic seems necessary and fine!

Returning the output of `_Raw.svd` directly doesn't work. When `computeUV` is false, the non-optional results `u` and `v` are undefined:

```swift
import TensorFlow
let x = Tensor<Float>(ones: [2, 2, 2])
print(_Raw.svd(x, computeUv: false))
```

```console
$ swift svd.swift
(s: [[1.9999999,       0.0],
 [1.9999999,       0.0]], u: 0.0, v: 0.0)
```

They seem to be zero tensors for this example.